home/templatetags/generic_components.py

```diff
@@ -78,6 +78,43 @@ def render_questionnaire_card(context, page, background_color=None, font_color=N
     return context
 
 
+@register.inclusion_tag('generic_components/page_link_page_card.html', takes_context=True)
+def render_page_link_page_card(context, page, background_color=None, font_color=None):
+    theme_settings = globals_.theme_settings
+
+    font_color = font_color or theme_settings.article_card_font_color
+    background_color = background_color or theme_settings.article_card_background_color
+    linked_page = page.get_page()
+
+    if isinstance(linked_page, Section):
+        template = 'generic_components/section_card.html'
+        context.update({
+            'section': linked_page,
+            'is_first_content': linked_page.is_first_content,
```

Comment:
it should be checked only for page link page and not for linked pages as linked page could be first content in other places